data-raw/dev/delays.R

```diff
@@ -0,0 +1,20 @@
+# Analysis of delay from public linelist
+
+library(data.table)
```

Comment:
i think its a really good idea to have the report delays as a package dataset extracted from the global linelist. Something like country, date, delay? This should really pull from the internet resource if possible.